dist/systemd/obsdodup.service

```diff
@@ -1,6 +1,7 @@
 [Unit]
 Description=OBS dodup, updates download on demand metadata
 After=network.target
+After=obsapisetup.service
```

Comment:
Nitpick: this could be added to the line before as:

`After=network.target obsapisetup.service`

Reply:
Thanks for the info, but I would like to keep it like it is because this is how it was merged into master. Otherwise a change of master and 2.10 would be required.